README.md

```diff
@@ -0,0 +1,78 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo-List
+
+> In this repository, we build a simple and todo list app. Look at the minimalist version we tried to build below:
+[![Capstone intro](https://img.youtube.com/vi/AcUd-_Yjjqg/0.jpg)](https://www.youtube.com/watch?v=AcUd-_Yjjqg)
+
+
+## Built With
+
+- HTML
+- CSS
+- ES6
+- Webpack
+- node/npm
+
+## Live Demo
+
+[https://adebsa2401.github.io/Todo-List](https://adebsa2401.github.io/Todo-List)
+
+## Getting Started
+
+<!-- **This is an example of how you may give instructions on setting up your project locally.**
```

Comment:
- [ ] In order to keep your code clean, it would be better to remove all code comments that don't describe the code itself.